packages/default/scss/common/_typography.scss

```diff
@@ -0,0 +1 @@
+@include exports( "common/typography" ) {}
```

Comment:
I'm still puzzled by placeholders. Why not add the file when it is needed?

Reply:
I will need it for the card / chat components.